airflow/www/templates/airflow/dags.html

```diff
@@ -133,7 +133,7 @@ <h2>DAGs</h2>
                 </td>
                 {# Column 5: Dag Schedule #}
                 <td>
-                  <a class="label label-default schedule" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
+                  <a class="label label-default schedule {{ dag.dag_id }}" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}">
```

Comment:
This can cause name conflicts. I think it's worth thinking about a different selector in this case, for example based on the `data-dag-id` attribute.
```html
<a class="label label-default schedule" href="{{ url_for('DagRunModelView.list') }}?_flt_3_dag_id={{ dag.dag_id }}" data-dag-id="{{ dag.dag_id }}">
```
```js
vaar dag_id = "my-dag-id"
document.querySelector('[data-dag-id="'+ dag_id+'"]')
```
See: https://issues.apache.org/jira/browse/AIRFLOW-6381

Reply:
Makes sense to me. And thanks for the snippet!

Will update shortly tonight and ping you for another review.